modules/lang-painless/src/main/java/org/elasticsearch/painless/ScriptInterface.java

```diff
@@ -35,6 +36,7 @@
 public class ScriptInterface {
     private final Class<?> iface;
     private final org.objectweb.asm.commons.Method executeMethod;
+    private final Definition.Type executeMethodReturnType;
     private final List<MethodArgument> arguments;
```

Comment:
Sorry for the nitpick: should this be named executeArguments to match the other members?